### DISCUSSION TITLE
Error during Sample 04.0: file transfer

### DISCUSSION THREAD

**Discussion by boxer-code** [eclipse-edc/Samples#17] [Q&A] [answered]
Hey everyone! I'm pretty new with using the EDC and I'm currently trying to run the samples. 
Sample 04.0 throws the following error and I don't know why: 
````
DEBUG 2022-06-21T13:29:28.2236664 [Provider] ContractNegotiation 5bbe480a-f95e-4312-962e-27df57ef795a is now in state CONFIRMED.
SEVERE 2022-06-21T13:29:56.969906 Fatal error processing transfer request: 95fb1acc-ab7b-44c2-beb6-dc9a8caf0001. Error details: Failed to delegate data transfer to Data Plane: Source file test-document.txt does not exist!
`````
Where do I need to add a file or what else am I missing in the setup? 
Thanks in advance!

**Comment by boxer-code** [comment, marked as answer]
I've found the error. It was caused by a conflict between my windows system and my wsl.

**Comment by tujinwei** [comment]
Hi,
can you share more details how you solved the conflict? I am stuck at he same problem. Thanks!
